ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -612,14 +612,10 @@ export const LeonardoAppLauncher = fp.flow(
         }
       }
 
-      const { workspace } = this.props;
+      // Terminal or SparkConsole LeoApplicationType
+
       this.incrementProgress(Progress.Creating);
-      const resp = await this.runtimeRetry(() =>
-        runtimeApi().localize(workspace.namespace, null, {
```

Comment:
this `null` was no longer acceptable.

Now it calls the standard method, which populates this parameter correctly.